notebooks/Tutorial_templates/tutorial_template_msb_v1.ipynb

```diff
@@ -0,0 +1,139 @@
+{
```

Comment:
<ul><li>Again, I'm not sure if we're putting this here to say that we always need Earthdata guidance written in this exact language. I would generalize this section as "Prerequisites" including an example of needing to include a reference to Earthdata Login. We may include tutorials that are not funded under the DAAC in the future, so this may not be relevant in all cases.</li><li>Spell check: authentification to authentication</li><li>I believe we should begin recommending the use of .netrc in our tutorials moving forward (though we can also demonstrate other optional auth methods including user input. This is coming up in the context of automated regression testing and is something we've been moving towards in our Openscapes tutorials as well.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>